Cargo.lock

```diff
@@ -195,6 +294,19 @@ dependencies = [
  "syn",
 ]
 
+[[package]]
+name = "env_logger"
+version = "0.8.3"
```

Comment:
We shouldn't nee to pull in env-logger... Is this coming from one of the new dependencies?

env-logger should only be needed by binary entry points and not by libraries. And for libraries, the `tracing` crate is definitely preferable.